src/main.cpp

```diff
@@ -15,8 +15,8 @@
 namespace po = boost::program_options;
 
 constexpr char window_title[] = "Lord of the Dumpsters: A Tommy Tooter Game";
-constexpr std::size_t window_width = 1280;
-constexpr std::size_t window_height = 720;
+constexpr std::size_t window_width =    950;
```

Comment:
Please don't commit this. It wouldn't be difficult to make a configuration file where you could add this locally. Or a command line switch, but I'd rather not have us going back and forth changing this.